universal/src/utils/small_string_benchmark.cpp

```diff
@@ -99,4 +99,44 @@ BENCHMARK(SmallString_Small_Move)
     ->Range(2, 2 << 10)
     ->Unit(benchmark::kMicrosecond);
 
+static void SmallString_Resize_and_Overwrite(benchmark::State& state) {
+  auto s = GenerateString(state.range(0));
```

Comment:
Append `Launder(std::move(string))` to the end of `GenerateString` to avoid the compiler optimizing away string size or something like that